docs/janssen-server/auth-server/endpoints/client-registration.md

```
@@ -555,6 +555,135 @@ SSA is validated based on `softwareStatementValidationType` which is enum.
+
+```json
+{
+  "loggingLevel": "DEBUG",
+  "loggingLayout": "text",
+  "externalLoggerConfiguration": "",
+  "enabledComponents": ["DCR"]
+}
+```
+
+This can be set via Config API:
+```bash
```

markdownlint-cli2 (0.22.0)

[warning] Fenced code blocks should be surrounded by blank lines

(MD031, blanks-around-fences)